README.md

```diff
@@ -19,14 +19,24 @@ So, the all-new goreleaser was born.
 
 ## Usage
 
-Basically, you need to create a `goreleaser.yml` file in the root of your
+```
+go get github.com/goreleaser/releaser
```

Comment:
If I'm correct, `goreleaser/get` downloads the latest version and runs it. That makes sense for CI, but not for a development machine. Shouldn't the preferred method be `go get` and `releaser`?